src/server/bnetserver/Server/Session.cpp

```diff
@@ -1141,7 +1144,11 @@ Battlenet::WoWRealm::ListUpdate* Battlenet::Session::BuildListUpdate(Realm const
         version << buildInfo->MajorVersion << '.' << buildInfo->MinorVersion << '.' << buildInfo->BugfixVersion << '.' << buildInfo->Build;
 
         listUpdate->Version = version.str();
-        listUpdate->Address = realm->GetAddressForClient(GetRemoteIpAddress());
+        // Prioritise ipv4, for compatibility
```

Comment:
In the case where this happens (ipv6 only worldserver), there isn't an ipv4 address available.  The client seems to handle it.